modules/statistics/templates/menu_statistics_dd_site.tpl

```diff
@@ -18,7 +18,7 @@
       <td>{$data[item].count}</td>
       {foreach from=$AllVisits item=visit name=VisitLoop}
       <td>{foreach from=$data[item].incompletes[$visit] item=Candidate name=CandLoop}
-            <a href="main.php?test_name={$Candidate.test_url}&candID={$Candidate.CandID}&sessionID={$Candidate.SessionID}&commentID={$Candidate.CommentID}">{$Candidate.PSCID}</a>
+            <a href="{$baseurl}/{$Candidate.test_url}/?candID={$Candidate.CandID}&sessionID={$Candidate.SessionID}&commentID={$Candidate.CommentID}">{$Candidate.PSCID}</a>
```

Comment:
I'm not sure if this is the most appropriate URL (I think baseURL/candID/sessionID/test_name/ might be cleaner), but this one should work too..